x-pack/plugins/apm/server/routes/alerts/rule_types/utils/get_interval_in_seconds.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getIntervalInSeconds } from './get_interval_in_seconds';
+
+describe('get lookback period in seconds for preview chart', () => {
+  it('returns lookback period when interval is 5 seconds', () => {
+    const lookback = getIntervalInSeconds('5s');
+    expect(lookback).toEqual(25);
+  });
```

Comment:
I think this function now mixes responsibilities. It is called `getIntervalInSeconds`. So for the interval `"5s` it should return `5`. It shouldn't take `INTERVAL_MULTIPLIER_FOR_LOOKBACK` into account

Reply:
Changed the function to only convert interval into seconds.